remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerImpl.java

```diff
@@ -95,6 +95,7 @@ void start() {
             getProtobufMetadataCacheConfig(globalConfiguration).build(),
             EnumSet.of(InternalCacheRegistry.Flag.USER, InternalCacheRegistry.Flag.PROTECTED, InternalCacheRegistry.Flag.PERSISTENT));
 
+      processSerializationContextInitializer(ServiceFinder.load(SerializationContextInitializer.class, globalConfiguration.classLoader()));
```

Comment:
Providing more detail on what I said before, here I was expecting the service loader would be used only if there are no manually configured SCI's in ```globalConfiguration.serialization()```.

The auto-registration is nice for simple use cases, which might account for 90% real use cases. But it cannot give you any way out of situations when schemas are conflicting for any reason.

To support debugging of such situations I propose adding INFO logging here, with the names of all SCI's being registred. And after the registration I would also dump the eventual errors, with ERROR level immediately.